Projects/Claims/Sources/Views/submitClaim/Checkout/SubmitClaimCheckoutNoRepairScreen.swift

```diff
@@ -88,50 +87,15 @@ public struct SubmitClaimCheckoutNoRepairScreen: View {
                     .foregroundColor(hLabelColor.primary)
                 Spacer()
 
-                if checkIfNotDecimal(value: model?.amount ?? 0) {
-
-                    hText(
-                        prefix + formatDoubleWithoutDecimal(value: model?.amount ?? 0) + " "
-                            + String(model?.currencyCode ?? "")
-                    )
-                    .foregroundColor(hLabelColor.secondary)
-                } else {
-                    hText(
-                        prefix + formatDoubleWithDecimal(value: model?.amount ?? 0) + " "
-                            + String(model?.currencyCode ?? "")
-                    )
-                    .foregroundColor(hLabelColor.secondary)
-                }
+                hText(
+                    prefix + (model?.formatAndPrintDouble(value: model?.amount ?? 0) ?? "") + " "
```

Comment:
Then after changes we should have hText(prefix + (model?.formatAndPrintDouble() ?? ""))